<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmailSentDialog</class>
 <widget class="QDialog" name="EmailSentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recover Password</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: white;</string>
  </property>
  <widget class="QLineEdit" name="code_input">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>170</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
 border: 1px solid grey;
 border-radius: 5px;
 padding: 5px; 
color: black;</string>
   </property>
   <property name="placeholderText">
    <string>Enter verification-code</string>
   </property>
  </widget>
  <widget class="QPushButton" name="verify_code_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>100</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #0078d7; color: white; border-radius: 10Px; font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Verify Code</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>391</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 248, 248);
border-radius: 10Px;</string>
   </property>
   <widget class="QLabel" name="label_title_5">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>220</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent; color: grey;</string>
    </property>
    <property name="text">
     <string>Did not receive the email ?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color : blue;</string>
    </property>
    <property name="text">
     <string>Resend Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title_6">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>30</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent; color: black;</string>
    </property>
    <property name="text">
     <string>Email Sent</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_title_7">
    <property name="geometry">
     <rect>
      <x>-70</x>
      <y>60</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent; color: grey;</string>
    </property>
    <property name="text">
     <string>We have sent you an email at </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_title_8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent; color: black;</string>
    </property>
    <property name="text">
     <string>my-emailid@gmail.com</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_title_9">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>80</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent; color: grey;</string>
    </property>
    <property name="text">
     <string>Check your inbox and enter the Verification-Code</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color : blue;</string>
    </property>
    <property name="text">
     <string>Change Email Adress</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title_10">
    <property name="geometry">
     <rect>
      <x>56</x>
      <y>249</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent; color: grey;</string>
    </property>
    <property name="text">
     <string>Wrong Email Adress ?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ressources.qrc">:/img/mailIcon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
   </property>
   </widget>
  </widget>
  <zorder>widget</zorder>
  <zorder>code_input</zorder>
  <zorder>verify_code_button</zorder>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>